paper-checkbox-light.html

```diff
@@ -0,0 +1,263 @@
+        border-top: none;
+        border-left: none;
+        border-right-width: calc(2/15 * var(--calculated-paper-checkbox-size));
+        border-bottom-width: calc(2/15 * var(--calculated-paper-checkbox-size));
+        border-color: var(--paper-checkbox-checkmark-color, white);
+        -webkit-transform-origin: 97% 86%;
+        transform-origin: 97% 86%;
+        box-sizing: content-box; /* protect against page-level box-sizing */
+      }
+
+      :host([checked]) #checkmark {
+        display: block;
+      }
+
+      :host-context([dir="rtl"]) #checkmark {
+        -webkit-transform-origin: 50% 14%;
+        transform-origin: 50% 14%;
+      }
+
+      /* disabled state */
+
+      :host([disabled]) #checkbox {
+        opacity: 0.5;
+        border-color: var(--paper-checkbox-unchecked-color, var(--primary-text-color));
+      }
+
+      :host([disabled][checked]) #checkbox {
+        background-color: var(--paper-checkbox-unchecked-color, var(--primary-text-color));
+        opacity: 0.5;
+      }
+    </style>
+
+    <div id="checkbox">
+      <div id="checkmark"></div>
+    </div>
+  </template>
+
+  <script>
+    Polymer({
+      is: 'paper-checkbox-light',
+
+      behaviors: [
+        Polymer.PaperRippleBehavior
+      ],
+
+      hostAttributes: {
+        role: 'checkbox',
+        'aria-checked': false,
+        tabindex: 0
+      },
+
+      properties: {
+        checked: {
+          type: Boolean,
+          notify: true,
+          reflectToAttribute: true,
+          observer: '_checkedChanged'
+        },
+
+        disabled: {
+          type: Boolean,
+          reflectToAttribute: true,
+          observer: '_disabledChanged'
+        },
+      },
+
+      listeners: {
+        'tap': '_onTap',
+        'keypress': '_onKeypress',
+        'down': '_rippleDown',
+        'up': '_rippleUp',
+        'focus': '_rippleDown',
+        'blur': '_rippleUp'
+      },
+
+      attached: function() {
+        this.setAttribute('aria-checked', !!this.checked);
+        this.setAttribute('aria-disabled', !!this.disabled);
+      },
+
+      _onTap: function() {
+        if (!this.disabled) {
+          this.checked = !this.checked;
+        }
+      },
+
+      _onKeypress: function(event) {
+        if (event.key === ' ' || event.code === 'Space' || event.keyCode === 32) {
+          event.preventDefault();
+
+          if (!this.disabled) {
+            this.checked = !this.checked;
+          }
+        }
+      },
+
+      _createRipple: function() {
+        var ripple = Polymer.PaperRippleBehavior._createRipple.call(this);
+        ripple.center = true;
+        ripple.classList.add('circle');
+        return ripple;
+      },
+
+      _rippleDown: function() {
+        this.getRipple().downAction();
+      },
+
+      _rippleUp: function() {
+        this.getRipple().upAction();
+      },
+
+      _checkedChanged: function(checked) {
+        this.setAttribute('aria-checked', this.checked);
```

Comment:
([comment in other thread](https://github.com/PolymerElements/paper-checkbox/pull/145#discussion_r73958983))